pkg/sql/schemachanger/scplan/internal/opgen/opgen_secondary_index.go

```diff
@@ -28,6 +28,12 @@ func init() {
 						IsSecondaryIndex: true,
 					}
 				}),
+				emit(func(this *scpb.SecondaryIndex) *scop.MaybeAddSplitForIndex {
+					return &scop.MaybeAddSplitForIndex{
+						TableID: this.TableID,
+						IndexID: this.IndexID,
+					}
+				}),
```

Comment:
Should we be emitting this op when creating new primary indexes also?

Reply:
Good point this looks like a regression, let me add this in the primary index code path too.